tests/gis_tests/geoapp/tests.py

```diff
@@ -300,6 +300,13 @@ def test_crosses_lookup(self):
             0
         )
 
+    @skipUnlessDBFeature("supports_isvalid_lookup")
+    def test_is_valid_lookup(self):
```

Comment:
isvalid or is_valid? Not sure if there's any rationale about the existing discrepency (e.g. same_as vs. coveredby)